stock_valuation_fifo_lot/models/product_template.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    def get_product_accounts(self, fiscal_pos=None):
```

Comment:
I prefer to extend `_get_accounting_data_for_valuation()` of `stock.move` instead to avoid having related adjustments in different files.